src/components/Books-home.js

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 import Form from './Form';
 
-const books = [{
-  title: 'Fifty Shades of Blue',
-  author: 'Fabien',
-  id: 1,
-}];
-
-const Home = () => (
-  <div>
-    <ul>
-      {books.map((book) => (
-        <Book key={book.id} title={book.title} author={book.author} />
-      ))}
-    </ul>
-    <Form />
-  </div>
-);
+const Home = () => {
+  const books = useSelector((state) => state.books);
```

Comment:
List of books displays books from Redux store (by using `useSelector` function 🚀